bakery/src/scripts/link_extras.py

```diff
@@ -73,8 +73,9 @@ def match_canonical_book(canonical_ids, containing_books):
         )
     except StopIteration:
         raise Exception(
-            "Multiple containing books, no canonical match!\n"
-            + f"{containing_books}"
+            "Multiple containing books, no canonical match!\n" +
+            f"module: {module_uuid}\n" +
```

Comment:
I tried using this branch to debug the error with `col28330`, and I actually found that providing the faulty `node.attrib['href']` or legacy ID instead of the converted module UUID was more useful as it allowed me to quickly find the link the in the CNXML and also run the extras query. Would that be better to print here for our purposes? 🤷‍♂️